package.json

```
@@ -1092,6 +1092,28 @@
 				"title": "Compare with Selected",
 				"category": "IBM i"
 			},
+			{
+				"command": "code-for-ibmi.compareCurrentFileWithMember",
+				"enablement": "code-for-ibmi:connected",
+				"title": "Compare with member...",
+				"category": "IBM i"
+			},
+			{
+				"command": "code-for-ibmi.compareCurrentFileWithStream",
```

Please rename to 
```suggestion
				"command": "code-for-ibmi.compareCurrentFileWithStreamfile",
```

to be more consistent.